src/services/ProcessorService.js

```diff
@@ -25,6 +25,11 @@ async function _fetchSubmissionDetails (submissionId, v5ChallengeId, memberId) {
     throw new Error('SRMs are ignored')
   }
 
+  // Ignore TopCrowd challenges
+  if (challenge.timelineTemplateId === config.get('TOPCROWD_CHALLENGE_TEMPLATE_ID')) {
+    throw new Error('TopCrowd challenges are ignored')
```

Comment:
I realize this has been merged already, but why are we throwing an error here? If this is an expected condition and we are simply ignoring TopCrowd challenges, if anything, why not just log an `info` message? Errors will show up in the logs (I assume, having not seen the rest of the code), and if it's not really an error, they shouldn't.